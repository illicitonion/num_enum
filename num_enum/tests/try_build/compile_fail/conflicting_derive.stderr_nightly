error[E0119]: conflicting implementations of trait `num_enum::TryFromPrimitive` for type `Numbers`
 --> $DIR/conflicting_derive.rs:1:35
  |
1 | #[derive(num_enum::FromPrimitive, num_enum::TryFromPrimitive)]
  |          -----------------------  ^^^^^^^^^^^^^^^^^^^^^^^^^^ conflicting implementation for `Numbers`
  |          |
  |          first implementation here
  |
  = note: this error originates in the derive macro `num_enum::TryFromPrimitive` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0119]: conflicting implementations of trait `std::convert::TryFrom<u8>` for type `Numbers`
 --> $DIR/conflicting_derive.rs:1:35
  |
1 | #[derive(num_enum::FromPrimitive, num_enum::TryFromPrimitive)]
  |                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: conflicting implementation in crate `core`:
          - impl<T, U> TryFrom<U> for T
            where U: Into<T>;
  = note: this error originates in the derive macro `num_enum::TryFromPrimitive` (in Nightly builds, run with -Z macro-backtrace for more info)
