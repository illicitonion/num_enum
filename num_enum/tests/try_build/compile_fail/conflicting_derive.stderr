error[E0119]: conflicting implementations of trait `std::convert::TryFrom<u8>` for type `Numbers`:
 --> $DIR/conflicting_derive.rs:1:35
  |
1 | #[derive(num_enum::FromPrimitive, num_enum::TryFromPrimitive)]
  |                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: conflicting implementation in crate `core`:
          - impl<T, U> std::convert::TryFrom<U> for T
            where U: std::convert::Into<T>;
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0119]: conflicting implementations of trait `num_enum::TryFromPrimitive` for type `Numbers`:
 --> $DIR/conflicting_derive.rs:1:10
  |
1 | #[derive(num_enum::FromPrimitive, num_enum::TryFromPrimitive)]
  |          ^^^^^^^^^^^^^^^^^^^^^^^  -------------------------- first implementation here
  |          |
  |          conflicting implementation for `Numbers`
  |
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
